ts/src/profilers/profile-serializer.ts

```diff
@@ -170,7 +170,9 @@ function createSampleCountValueType(table: StringTable):
  */
 function createTimeValueType(table: StringTable): perftools.profiles.ValueType {
   return new perftools.profiles.ValueType({
-    type: table.getIndexOrAdd('time'),
+    // TODO: change type back to time.
+    // This is a work-around so profiles can be viewed using the UI.
+    type: table.getIndexOrAdd('wall'),
```

Comment:
I do not like this Github feature where it's not possible to comment on lines that were not changed in the PR, but several lines above you add "samples" metric. Java agent names that "sample", in singular, so let's change to "sample" here as well.

sample

Reply:
Done!